docs/index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
```

Comment:
what is the usage of this file ?
I expect to see see an index in the form of .md file so it can be normally  view via github